### PR TITLE
Issue/10634 material pages list

### LINKED ISSUE #10634: Implement Material theme on Pages List screen
We should switch widgets used at Pages LIst screen to their material counterparts, remove color references where possible, and switch to using material `textAppearance` attributes.

### PR DESCRIPTION
**Pull request by khaykov**
Fixes #10634 

This PR moves Pages screen to Material Theme and adds Dark Theme support. Changes are cosmetic.

At the time, not all screens have Dark Theme support, so the layout will look broken at most of the places.

comparison 

To test:
- Navigate to Pages screen and make sure it looks as expected in both light and dark themes. The theme can be changed from App Settings.

This PR requires one designer and one developer to review it.

PR submission checklist:

- [x] I have considered adding unit tests where possible.

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by mattmiklic** [comment]
This is looking good to me. 👍

**Comment by khaykov** [comment]
Updated the style of the primary button (used in empty view) after discussion with @mattmiklic .

Screenshot_1575928168

File: WordPress/src/main/java/org/wordpress/android/ui/pages/PageItemViewHolder.kt [hunk 1]
```diff
@@ -163,10 +162,6 @@ sealed class PageItemViewHolder(internal val parent: ViewGroup, @LayoutRes layou
                 radioButton.setOnClickListener {
                     onParentSelected(pageItem)
                 }
-
-                @Suppress("DEPRECATION")
-                CompoundButtonCompat.setButtonTintList(radioButton,
```
Comment (planarvoid) [inline WordPress/src/main/java/org/wordpress/android/ui/pages/PageItemViewHolder.kt]: when we remove this, the radio button has the light blue color, right? At least that's what see in the app now. Is that what we want? 😄 
Screenshot 2019-12-11 at 14 44 00

Comment (khaykov) [reply WordPress/src/main/java/org/wordpress/android/ui/pages/PageItemViewHolder.kt]: Yes, there is no reason for this radio buttons to be special ones :)

**Comment by khaykov** [comment]
Thank you for taking a look, @planarvoid ! Yes, removal of a grey background is part of the plan :)

**Comment by khaykov** [comment]
@planarvoid Since you only commented on visual changes (and all of them are intentional) I'm going to merge this PR since couple of other ones depend on it :)

Again, thanks for taking the time to check it!

**Pull request merged**